pkg/migrate/migrate_test.go

```diff
@@ -692,18 +696,20 @@ func TestGetPVCs(t *testing.T) {
 					corev1.ReadWriteOnce,
 				}, pvc.Spec.AccessModes)
 			},
-			originalPVCs: map[string][]corev1.PersistentVolumeClaim{
+			originalPVCs: map[string][]pvcCtx{
 				"ns1": {
 					{
-						ObjectMeta: metav1.ObjectMeta{
-							Name:      "pvc1",
-							Namespace: "ns1",
-						},
-						Spec: corev1.PersistentVolumeClaimSpec{
-							VolumeName: "pv1",
-							AccessModes: []corev1.PersistentVolumeAccessMode{
-								corev1.ReadWriteMany,
-								corev1.ReadWriteOnce,
+						claim: &corev1.PersistentVolumeClaim{
+							ObjectMeta: metav1.ObjectMeta{
+								Name:      "pvc1",
+								Namespace: "ns1",
+							},
+							Spec: corev1.PersistentVolumeClaimSpec{
+								VolumeName: "pv1",
+								AccessModes: []corev1.PersistentVolumeAccessMode{
+									corev1.ReadWriteMany,
```

Comment:
localpv doesnt support rwx volumes. should we handle that case with an error somewhere?

Reply:
We should.

Reply:
I'd prefer to handle it by detecting that the migration pod or pvc creation code have failed (rather than special casing localpv)

Reply:
that makes sense to me. lets create an issue and follow up after

Reply:
Tracking issue: https://app.shortcut.com/replicated/story/61144/check-pvc-accessmode-when-migrating-pvcs-using-pvmigrate